lua/lualine/utils/utils.lua

```diff
@@ -54,4 +54,11 @@ function M.clear_highlights()
   end
 end
 
+function M.get_short_cwd()
+  if os.getenv('HOME') == os.getenv('PWD') then
+    return '~/'
+  end
+  return os.getenv('PWD'):gsub(os.getenv('HOME'), '~')
```

Comment:
Like I said before there's no need for this function . 
```lua
vim.fn.fnamemodify('long_home/long_path', ':~')
```

Reply:
you are right, I forgot about that, thanks for pointing that out